README.md

```diff
@@ -114,14 +114,19 @@ dependency can be provided two ways:
 If you do not wish to build and run the tests, add `-DBENCHMARK_ENABLE_GTEST_TESTS=OFF`
 to `CMAKE_ARGS`.
 
+### Speeding up the build
+
+If you find the build or test runs to be slow, you can pass `--parallel` to the
+`cmake` and `ctest` commands as per the cmake documentation.
+
 ### Debug vs Release
 
 By default, benchmark builds as a debug library. You will see a warning in the
-output when this is the case. To build it as a release library instead, add
-`-DCMAKE_BUILD_TYPE=Release` when generating the build system files, as shown
-above. The use of `--config Release` in build commands is needed to properly
-support multi-configuration tools (like Visual Studio for example) and can be
-skipped for other build systems (like Makefile).
+output when this is the case. To build it as a release library instead, as in
+the above instructions, add `--config Release` to the cmake commands. The use
+of `--config Release` in build commands is needed to properly support
+multi-configuration tools (like Visual Studio for example) and can be skipped
+for other build systems (like Makefile).
```

Comment:
The change you did here is not really correct.

When configuring CMake (the first call to `cmake` you do) you have to set the configuration by providing the `CMAKE_BUILD_TYPE`, which you do by adding `-DCMAKE_BUILD_TYPE=...` (or in some more verbose forms like for example `-DCMAKE_BUILD_TYPE:STRING="..."`).

The configuration command doesn't support the `--config` argument.